{
  "type": "issue",
  "issue": {
    "id": 2823509936,
    "node_id": "I_kwDOABII586oS1ew",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/31770",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/31770/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/31770/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/31770/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/31770",
    "number": 31770,
    "state": "open",
    "state_reason": null,
    "title": "fuzz: oss-fuzz coverage build is failing",
    "body": "It first failed on the 25th (https://oss-fuzz-build-logs.storage.googleapis.com/log-7da28be4-dcf5-4e23-a6ca-153f67ad2f60.txt):\n```bash\nStep #7: \u001b[0m\u001b[0;31mwarning: The file '/usr/evmap.c' isn't covered.\nStep #7: \u001b[0m\u001b[0;31merror: /workspace/out/libfuzzer-coverage-x86_64/usr/arc4random.c: No such file or directory\nStep #7: \u001b[0m\u001b[0;31mwarning: The file '/usr/arc4random.c' isn't covered.\nStep #7: \u001b[0m\u001b[0;31merror: /workspace/out/libfuzzer-coverage-x86_64/usr/buffer.c: No such file or directory\nStep #7: \u001b[0m\u001b[0;31mwarning: The file '/usr/buffer.c' isn't covered.\nStep #7: \u001b[0m\u001b[0;31merror: /workspace/out/libfuzzer-coverage-x86_64/usr/util-internal.h: No such file or directory\nStep #7: \u001b[0m\u001b[0;31mwarning: The file '/usr/util-internal.h' isn't covered.\nStep #7: \u001b[0m[2025-01-31 07:01:44,276 DEBUG] Finished generating per-file code coverage summary.\nStep #7: [2025-01-31 07:01:44,277 DEBUG] Generating file view html index file as: \"/workspace/out/libfuzzer-coverage-x86_64/report/linux/file_view_index.html\".\nStep #7: Traceback (most recent call last):\nStep #7:   File \"/opt/code_coverage/coverage_utils.py\", line 829, in <module>\nStep #7:     sys.exit(Main())\nStep #7:   File \"/opt/code_coverage/coverage_utils.py\", line 823, in Main\nStep #7:     return _CmdPostProcess(args)\nStep #7:   File \"/opt/code_coverage/coverage_utils.py\", line 780, in _CmdPostProcess\nStep #7:     processor.PrepareHtmlReport()\nStep #7:   File \"/opt/code_coverage/coverage_utils.py\", line 577, in PrepareHtmlReport\nStep #7:     self.GenerateFileViewHtmlIndexFile(per_file_coverage_summary,\nStep #7:   File \"/opt/code_coverage/coverage_utils.py\", line 450, in GenerateFileViewHtmlIndexFile\nStep #7:     self.GetCoverageHtmlReportPathForFile(file_path),\nStep #7:   File \"/opt/code_coverage/coverage_utils.py\", line 422, in GetCoverageHtmlReportPathForFile\nStep #7:     assert os.path.isfile(\nStep #7: AssertionError: \"/usr/arc4random.c\" is not a file.\nStep #7: ********************************************************************************\nStep #7: Code coverage report generation failed.\nStep #7: To reproduce, run:\nStep #7: python infra/helper.py build_image bitcoin-core\nStep #7: python infra/helper.py build_fuzzers --sanitizer coverage bitcoin-core\nStep #7: python infra/helper.py coverage bitcoin-core\nStep #7: ********************************************************************************\nFinished Step #7\nERROR\nERROR: build step 7 \"gcr.io/oss-fuzz-base/base-runner\" failed: step exited with non-zero status: 1\n```\n\nSee also: https://issues.oss-fuzz.com/issues/391974928.",
    "user": {
      "login": "fanquake",
      "id": 863730,
      "node_id": "MDQ6VXNlcjg2MzczMA==",
      "avatar_url": "https://avatars.githubusercontent.com/u/863730?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/fanquake",
      "html_url": "https://github.com/fanquake",
      "followers_url": "https://api.github.com/users/fanquake/followers",
      "following_url": "https://api.github.com/users/fanquake/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/fanquake/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/fanquake/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/fanquake/subscriptions",
      "organizations_url": "https://api.github.com/users/fanquake/orgs",
      "repos_url": "https://api.github.com/users/fanquake/repos",
      "events_url": "https://api.github.com/users/fanquake/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/fanquake/received_events",
      "type": "User",
      "site_admin": false,
      "patch_url": null
    },
    "labels": [],
    "assignees": [],
    "author_association": "MEMBER",
    "locked": false,
    "comments": 0,
    "created_at": "2025-01-31T13:33:00Z",
    "updated_at": "2025-01-31T13:33:00Z"
  },
  "events": []
}